include/oneapi/dpl/dynamic_selection

```diff
@@ -0,0 +1,46 @@
+// -*- C++ -*-
```

Comment:
General question about all files. What about uglification?

Reply:
Maybe it will make sense to consider later, when this stuff will be integrated more into the library. For now, let's not bother.